src/components/Capture/capture.js

```diff
@@ -85,6 +91,11 @@ class Capture extends Component {
     })
   }
 
+  onVideoRecorded = (blob) => {
+    const url = window.URL.createObjectURL(blob);
```

Comment:
`createObjectURL` needs a `revokeObjectURL()` when it's no longer needed. This can be added in a separate ticket, but we cannot forget it.

Reply:
Yes! I was thinking about that late on Friday!